package.json

```diff
@@ -18,26 +18,30 @@
     "p-whilst": "^1.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-syntax-object-rest-spread": "^7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/preset-env": "^7.2.0",
+    "@babel/runtime": "^7.2.0",
     "@orbitdb/eslint-config-orbitdb": "0.0.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^8.0.2",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "datastore-level": "~0.8.0",
-    "go-ipfs-dep": "~0.4.17",
-    "ipfs": "~0.31.4",
-    "ipfs-repo": "~0.23.1",
-    "ipfsd-ctl": "~0.39.1",
+    "babel-loader": "^8.0.4",
+    "datastore-level": "^0.10.0",
```

Comment:
Note that we use `~` instead of `^` to avoid pulling in minor version updates (this is to avoid automatically using the latest version which may contain breaking changes).

These should be changed back to `~` (this and others below the line).